src/frontend/src/flows/recovery/confirmSeedPhrase.ts

```diff
@@ -43,12 +43,22 @@ const confirmSeedPhraseTemplate = ({
     if (word.check) {
       const elem: Ref<HTMLInputElement> = createRef();
       // NOTE: typescript can't follow if word is deconstructed with {...word}
-      return { word: word.word, check: word.check, elem };
+      return {
+        word: word.word,
+        check: word.check,
+        elem,
+      };
```

Comment:
did something change here? are you running a different version of prettier?

Reply:
I used `npm run format` as always. My editor thought it was a great idea to put it on new lines after I have removed the property I had added earlier. Apparently prettier likes both.